components/buttons/DefaultButton.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component} from 'react';
+import {Button, ButtonGroup} from 'react-bootstrap';
+
+import '../../less/buttons.less';
+import '../../less/theme.less';
+
+const DefaultButton = () => (
```

Comment:
It might be a good idea to still create files for them under components which simply export the react-bootstrap component. That way, if we ever do need to add anything to the component, we can extend it without MB having to know or change its imports.

```JavaScript
// components/Button.js?

import {Button} from 'react-bootstrap';

// We can change the export later if we need to
// change the button HTML for some reason.
export default Button;
```

And it's probably still nice to see how they look in storybook, even if we're not customizing them beyond the CSS, right?

Reply:
Yes, let's have all components in storybook, even if we are not customizing it. :)
Added a file which simply exports the react-bootstrap component as well. I think this should work just fine!